generators/csharp/base/src/asIs/Pager.Template.cs

```diff
@@ -455,6 +455,11 @@ SetCursor setCursor
         };
     }
 
+    private static bool isCursorNull(TCursor cursor)
+    {
+        return (cursor is null || (cursor is string s && string.IsNullOrEmpty(s)));
```

Comment:
Since you already check for null, using `string.IsNullOrEmpty(s)` has some redundance.
You can simply check `s == ""`.
What I'd probably try go for is this:

```csharp
return request switch
        {
            null or "" => true,
            _ => false
        };
```

and rename `isCursorNull` to `isCursorNullOrEmpty`